src/node/routes/login.ts

```diff
@@ -98,4 +88,35 @@ router.post("/", async (req, res) => {
   } catch (error) {
     res.send(await getRoot(req, error))
   }
+}
+
+export const router = Router()
+
+router.use((req : any, res : any, next : any) => {
```

Comment:
```suggestion
router.use((req: Request, res: Response, next: NextFunction) => {
```

For this to work you'll need to import `NextFunction` from `express` at the top of the file.